### PR TITLE
New design for Brands & Suppliers List

### LINKED ISSUE #9670: [BOOM-5316] Brand list module
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5316).

- _**Reporter:**_ marion_francois
- _**Created at:**_ Fri, 13 Apr 2018 16:55:55 +0200

<p>In brand list module, I can choose to display brands in plain text or drop-down.</p>

<p>If I choose drop-down, it's not very pretty</p>
        
capture du 2018-04-13 16-54-35

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | New deisgn for Brands & Suppliers List
| Type?         | bug fix
| Category?     | FO 
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #9670 #9669
| How to test?  | In brand list  or supplier list module, choose to display elements in plain text or drop-down and go on FO and check a category page.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13460)

### CODE REVIEW

**Comment by rdy4ever** [comment]
Should we really add a 67 kb (minified) javascript library (bundled into theme.js - thus loaded on each page) only to be used on category pages - just because regular select boxes "are not pretty"?
Considering that currently theme.js weights 170 kb, adding 67 kb to that would mean a 40% increase. This is a no no performance-wise.

EDIT: somehow you added 106 kb to theme.js, going from 170 kb to 276 kb.

File: themes/classic/_dev/js/theme.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ import 'expose-loader?Tether!tether';
 import 'bootstrap/dist/js/bootstrap.min';
 import 'flexibility';
 import 'bootstrap-touchspin';
+import 'select2';
+import 'select2/dist/css/select2.min.css';
```
Comment (rdy4ever) [inline themes/classic/_dev/js/theme.js:30]: Are you really importing a css file into theme.js? :)

Comment (PierreRambaud) [reply themes/classic/_dev/js/theme.js:30]: Don't worry, these files are compiled with webpack and extract from js configuration to a proper css files ;)

**Comment by PierreRambaud** [comment]
Agree with @rdy4ever  it's really overkill and select must be improve with the ui-kit instead.
PIng @eternoendless wdyt?

**Comment by Quetzacoalt91** [comment]
The ui-kit is currently for the back office. As far as I know we don't use it on the front office.

However, we already have solutions for having prettier selects:

Capture d’écran du 2019-05-01 10-41-29

I guess reusing the same idea would be enough.

**Comment by Quetzacoalt91** [comment]
Regarding the given reviews, I would rather close this PR and wait for another solution.

Thanks for your understanding @YeLnatSs

**Comment by YeLnatSs** [comment]
@Quetzacoalt91 Okay I understand very well.
Thank you

**Comment by rdy4ever** [comment]
> The ui-kit is currently for the back office. As far as I know we don't use it on the front office.
> 
> However, we already have solutions for having prettier selects:
> 
> Capture d’écran du 2019-05-01 10-41-29
> 
> I guess reusing the same idea would be enough.

It actually uses the same design if you use Faceted search module:
image

**Pull request closed without merging**